src/components/MessageInput/styles.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../lib/commonStyles/colors';
+@import '../../lib/commonStyles/full-size';
+
+.root {
+  position: absolute;
+  bottom: 0px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 2px 2px 2px;
+  font-family: Helvetica;
+  border-top: 1px solid #e3e3e5;
+  box-sizing: border-box;
+}
+
+.textField {
+  margin-right: 115px;
+  box-sizing: border-box;
+
+  textarea {
+    background: transparent;
+    font-size: 13px;
+    line-height: 18px;
+    width: 100%;
+    margin: 10px 0 10px 10px;
+    resize: none;
+    border: none;
+    outline: medium none!important;
+    box-sizing: border-box;
+
+    &::placeholder {
+      color: #a9a9a9;
+    }
+  }
+}
+
+.submitField {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  width: 115px;
+  margin-top: -18px;
+  margin-right: 2px;
+}
+
+.sendButton {
+  display: block;
+  margin: 0 auto;
+  width: 73px;
+  height: 36px;
+  box-sizing: border-box;
+  font-size: 13px;
+  line-height: 18px;
+  padding: 7px;
+  border-radius: 18px;
+  outline: medium none!important;
+  color: #fff;
+  background-color: $primary-color;
+  border-style: none;
+}
+
+.sendButton:hover {
+  background-color: $primary-color-highlight;
+  color: #fff;
+}
+
+.sendButton:disabled,
+.sendButton:disabled:hover {
+  background-color: #fff!important;
+  border: #c9c9c9 1px solid;
+  color: #a9a9a9;
```

Comment:
Do we have color variables for disabled button?

Reply:
hmm... I am not sure why the !important was added in the first place. Let's keep it there just in case there were some important reasons to do so.